README.md
# lernstick-usertemplate-winterthur
user template customizations for Winterthur


debian/changelog
lernstick-usertemplate-winterthur (65) lernstick-9-winterthur; urgency=medium

  * [483bf2] switched from Master-PDF-Editor v4 to v5 as default app for PDFs

 -- Ronny Standtke <ronny.standtke@gmx.net>  Thu, 24 Jan 2019 14:25:24 +0100

lernstick-usertemplate-winterthur (64) lernstick-9-winterthur; urgency=medium

  * [8c529a] added handler for Citrix Receiver in Firefox default profile

 -- Ronny Standtke <ronny.standtke@gmx.net>  Sat, 15 Dec 2018 15:52:23 +0100

lernstick-usertemplate-winterthur (63) lernstick-9-winterthur; urgency=medium

  * [2c24bc] updated Firefox profile (always activate Flash, 80% zoom for Lehrmittelverlag Zürich

 -- Ronny Standtke <ronny.standtke@gmx.net>  Fri, 14 Dec 2018 00:01:13 +0100

lernstick-usertemplate-winterthur (62) lernstick-9-winterthur; urgency=medium

  * [9ffb1d] replaced MS Word with LibreOffice in GNOME favorites
  * [f5e027] updated dash-to-dock settings
  * [724689] removed MS Office 2013 from mimeapps list
  * [cbb07e] added config for nautilus zoom level

 -- Ronny Standtke <ronny.standtke@gmx.net>  Thu, 13 Dec 2018 23:37:45 +0100

lernstick-usertemplate-winterthur (61) lernstick-9-winterthur; urgency=medium

  * [60fbd9] moved dash to the right side of the screen
             (prevents collisions with desktop icons)

 -- Ronny Standtke <ronny.standtke@gmx.net>  Sun, 14 Oct 2018 13:40:28 +0200

lernstick-usertemplate-winterthur (60) lernstick-9-winterthur; urgency=medium

  * [249ef0] set power-button-action to 'nothing'

 -- Ronny Standtke <ronny.standtke@gmx.net>  Tue, 19 Jun 2018 19:02:22 +0200

lernstick-usertemplate-winterthur (59) lernstick-9-winterthur; urgency=medium

  * [4b3767] moved fsview in GNOME from app folder "utilities" to "system"

 -- Ronny Standtke <ronny.standtke@gmx.net>  Mon, 11 Jun 2018 22:23:40 +0200

lernstick-usertemplate-winterthur (58) lernstick-9-winterthur; urgency=medium

  * [e8bd09] updated GNOME backgrounds

 -- Ronny Standtke <ronny.standtke@gmx.net>  Mon, 11 Jun 2018 21:24:41 +0200

lernstick-usertemplate-winterthur (57) lernstick-9-winterthur; urgency=medium

  * [76e775] added some document templates

 -- Ronny Standtke <ronny.standtke@gmx.net>  Sat, 09 Jun 2018 21:18:04 +0200

lernstick-usertemplate-winterthur (56) lernstick-9-winterthur; urgency=medium

  * [44fb7a] default to Office 2013

 -- Ronny Standtke <ronny.standtke@gmx.net>  Sun, 06 May 2018 23:46:52 +0200

lernstick-usertemplate-winterthur (55) lernstick-9-winterthur; urgency=medium

  * [943d69] upgraded to MS Office 2013

 -- Ronny Standtke <ronny.standtke@gmx.net>  Sun, 06 May 2018 22:03:37 +0200

lernstick-usertemplate-winterthur (54) lernstick-9-winterthur; urgency=medium

  * [a544de] updated GNOME app folders for Debian 9

 -- Ronny Standtke <ronny.standtke@gmx.net>  Sat, 05 May 2018 11:17:55 +0200

lernstick-usertemplate-winterthur (53) lernstick-9-winterthur; urgency=medium

  * [b43860] updated app folder info for Debian 9

 -- Ronny Standtke <ronny.standtke@gmx.net>  Sun, 22 Apr 2018 20:16:30 +0200

lernstick-usertemplate-winterthur (52) lernstick-8-winterthur; urgency=medium

  * [aef382] added icedove->thunderbird migration assisstant to GNOME's app folder "Network"

 -- Ronny Standtke <ronny.standtke@gmx.net>  Thu, 30 Nov 2017 00:24:44 +0100

lernstick-usertemplate-winterthur (51) lernstick-8-winterthur; urgency=medium

  * [8d1965] added gthumb to GNOMEs app folder AudioVideo

 -- Ronny Standtke <ronny.standtke@gmx.net>  Wed, 20 Sep 2017 15:09:52 +0200

lernstick-usertemplate-winterthur (50) lernstick-8-winterthur; urgency=medium

  * [95f42b] disabled update checks in Master PDF Editor

 -- Ronny Standtke <ronny.standtke@gmx.net>  Sun, 14 May 2017 13:51:45 +0200

lernstick-usertemplate-winterthur (49) lernstick-8-winterthur; urgency=medium

  * [78397a] updated firefox profile (prefer German language on websites)

 -- Ronny Standtke <ronny.standtke@gmx.net>  Thu, 06 Apr 2017 13:07:31 +0200

lernstick-usertemplate-winterthur (48) lernstick-8-winterthur; urgency=medium

  * [dde80e] made masterpdfeditor the default application for pdf files

 -- Ronny Standtke <ronny.standtke@gmx.net>  Wed, 15 Mar 2017 15:35:47 +0100

lernstick-usertemplate-winterthur (47) lernstick-8-winterthur; urgency=medium

  * [a10410] moved "rechenblatt" to different app folders

 -- Ronny Standtke <ronny.standtke@gmx.net>  Sat, 11 Mar 2017 14:04:34 +0100

lernstick-usertemplate-winterthur (46) lernstick-8-winterthur; urgency=medium

  * [4537a2] switched from firefox to firefox-esr

 -- Ronny Standtke <ronny.standtke@gmx.net>  Sat, 11 Mar 2017 13:21:48 +0100

lernstick-usertemplate-winterthur (45) lernstick-8-winterthur; urgency=medium

  * [bcd731] updated gnome apps folders

 -- Ronny Standtke <ronny.standtke@gmx.net>  Mon, 27 Feb 2017 10:10:30 +0100

lernstick-usertemplate-winterthur (44) lernstick-8-winterthur; urgency=medium

  * [f64d01] added default config (German) for Master PDF Editor

 -- Ronny Standtke <ronny.standtke@gmx.net>  Sun, 26 Feb 2017 10:46:54 +0100

lernstick-usertemplate-winterthur (43) lernstick-8-winterthur; urgency=medium

  * [b6e177] updated favorite apps
  * [75b81f] show trash icon on desktop
  * [c2d03f] added default passwordless keyring

 -- Ronny Standtke <ronny.standtke@gmx.net>  Sat, 25 Feb 2017 09:23:02 +0100

lernstick-usertemplate-winterthur (42) lernstick-8-winterthur; urgency=medium

  * [580fe7] added default config for firefox and chromium

 -- Ronny Standtke <ronny.standtke@gmx.net>  Fri, 24 Feb 2017 17:11:06 +0100

lernstick-usertemplate-winterthur (41) lernstick-8-winterthur; urgency=medium

  * [80aac3] added Young World desktop files for teacher preparation

 -- Ronny Standtke <ronny.standtke@gmx.net>  Wed, 04 Jan 2017 16:14:42 +0100

lernstick-usertemplate-winterthur (40) lernstick-8-winterthur; urgency=medium

  * [0ca4c0] added display config to GNOME favorites

 -- Ronny Standtke <ronny.standtke@gmx.net>  Thu, 15 Sep 2016 17:39:09 +0200


etc/skel/.mozilla/firefox/ypwua0kj.default/sessionstore.js
{"version":["sessionrestore",1],"windows":[{"tabs":[{"entries":[{"url":"http://www.schucom.ch/lernstick/","title":"Startseite - Lernplattform zum Lernstick","charset":"UTF-8","ID":1,"docshellID":5,"originalURI":"http://www.schucom.ch/lernstick/","triggeringPrincipal_base64":"SmIS26zLEdO3ZQBgsLbOywAAAAAAAAAAwAAAAAAAAEY=","docIdentifier":1,"persist":true}],"lastAccessed":1491476419205,"hidden":false,"attributes":{},"userContextId":0,"index":1,"image":"http://www.schucom.ch/lernstick/favicon.ico","iconLoadingPrincipal":"ZT4OTT7kRfqycpfCC8AeuAAAAAAAAAAAwAAAAAAAAEYB3pRy0IA0EdOTmQAQS6D9QJIHOlRteE8wkTq4cYEyCMYAAAAC/////wAAAFABAAAAIGh0dHA6Ly93d3cuc2NodWNvbS5jaC9sZXJuc3RpY2svAAAAAAAAAAQAAAAHAAAADgAAAAf/////AAAAB/////8AAAAHAAAADgAAABUAAAALAAAAFQAAAAsAAAAVAAAACwAAACAAAAAAAAAAIP////8AAAAA/////wAAABX/////AAAAFf////8BAAAAAAAAAAAAAQAAAAAAAA=="}],"selected":1,"_closedTabs":[],"width":1440,"height":880,"screenX":238,"screenY":79,"sizemode":"normal","title":"Startseite - Lernplattform zum Lernstick","_shouldRestore":true,"closedAt":1491476419214,"closedId":0}],"selectedWindow":0,"_closedWindows":[],"session":{"lastUpdate":1491476419358,"startTime":1491476393262,"recentCrashes":0},"global":{}}

etc/skel/.mozilla/firefox/ypwua0kj.default/sessionstore-backups/previous.js
{"version":["sessionrestore",1],"windows":[{"tabs":[{"entries":[{"url":"about:home","title":"Mozilla Firefox-Startseite","charset":"","ID":2,"docshellID":5,"docIdentifier":2,"persist":true}],"lastAccessed":1466876184848,"hidden":false,"attributes":{},"userContextId":0,"index":1,"image":"chrome://branding/content/icon32.png"}],"selected":1,"_closedTabs":[{"state":{"entries":[{"url":"about:preferences#advanced","title":"Einstellungen","charset":"","ID":5,"docshellID":10,"originalURI":"about:preferences#advanced","owner_b64":"SmIS26zLEdO3ZQBgsLbOywAAAAAAAAAAwAAAAAAAAEY=","docIdentifier":4,"structuredCloneState":"DAAAgAQA//9wYW5lQWR2YW5jZWQAAAAA","structuredCloneVersion":6,"persist":true}],"lastAccessed":1466876179871,"hidden":false,"attributes":{},"userContextId":0,"index":1,"image":"chrome://browser/skin/preferences/in-content/favicon.ico"},"title":"Einstellungen","image":"chrome://browser/skin/preferences/in-content/favicon.ico","pos":1,"closedAt":1466876179871}],"width":"1440","height":"1069","screenX":"134","screenY":"87","sizemode":"maximized","title":"Mozilla Firefox-Startseite","_shouldRestore":true,"closedAt":1466876184854}],"selectedWindow":0,"_closedWindows":[],"session":{"lastUpdate":1466876184982,"startTime":1466876137819,"recentCrashes":0},"global":{}}

etc/skel/.mozilla/firefox/ypwua0kj.default/prefs.js
// Mozilla User Preferences

// DO NOT EDIT THIS FILE.
//
// If you make changes to this file while the application is running,
// the changes will be overwritten when the application exits.
//
// To change a preference value, you can either:
// - modify it via the UI (e.g. via about:config in the browser); or
// - set it within a user.js file in your profile.

user_pref("app.normandy.first_run", false);
user_pref("app.shield.optoutstudies.enabled", false);
user_pref("app.update.lastUpdateTime.addon-background-update-timer", 1487950036);
user_pref("app.update.lastUpdateTime.blocklist-background-update-timer", 1487950156);
user_pref("app.update.lastUpdateTime.browser-cleanup-thumbnails", 1491476310);
user_pref("app.update.lastUpdateTime.experiments-update-timer", 1487949916);
user_pref("app.update.lastUpdateTime.search-engine-update-timer", 1544741723);
user_pref("app.update.lastUpdateTime.xpi-signature-verification", 1487951209);
user_pref("browser.cache.disk.capacity", 358400);
user_pref("browser.cache.disk.filesystem_reported", 1);
user_pref("browser.cache.disk.smart_size.first_run", false);
user_pref("browser.cache.disk.smart_size.use_old_max", false);
user_pref("browser.cache.frecency_experiment", 1);
user_pref("browser.download.importedFromSqlite", true);
user_pref("browser.download.panel.shown", true);
user_pref("browser.laterrun.bookkeeping.profileCreationTime", 1466876090);
user_pref("browser.laterrun.bookkeeping.sessionCount", 4);
user_pref("browser.migration.version", 66);
user_pref("browser.newtabpage.activity-stream.impressionId", "{88481369-a9d2-40d4-8d3b-1b15c77c01d8}");
user_pref("browser.newtabpage.activity-stream.migrationExpired", true);
user_pref("browser.newtabpage.enhanced", false);
user_pref("browser.newtabpage.storageVersion", 1);
user_pref("browser.onboarding.seen-tourset-version", 2);
user_pref("browser.onboarding.tour-type", "update");
user_pref("browser.pageActions.persistedActions", "{\"version\":1,\"ids\":[\"bookmark\",\"bookmarkSeparator\",\"copyURL\",\"emailLink\",\"sendToDevice\",\"pocket\",\"screenshots\"],\"idsInUrlbar\":[\"pocket\",\"bookmark\"]}");
user_pref("browser.pagethumbnails.storage_version", 3);
user_pref("browser.places.importBookmarksHTML", false);
user_pref("browser.places.smartBookmarksVersion", 8);
user_pref("browser.preferences.advanced.selectedTabIndex", 1);
user_pref("browser.reader.detectedFirstArticle", true);
user_pref("browser.rights.3.shown", true);
user_pref("browser.safebrowsing.provider.google.lastupdatetime", "1491476208499");
user_pref("browser.safebrowsing.provider.google.nextupdatetime", "1491477968499");
user_pref("browser.safebrowsing.provider.mozilla.lastupdatetime", "1544741695900");
user_pref("browser.safebrowsing.provider.mozilla.nextupdatetime", "1544745295900");
user_pref("browser.search.countryCode", "CH");
user_pref("browser.search.region", "CH");
user_pref("browser.search.widget.inNavBar", true);
user_pref("browser.sessionstore.upgradeBackup.latestBuildID", "20181002222838");
user_pref("browser.shell.mostRecentDateSetAsDefault", "1544741890");
user_pref("browser.slowStartup.averageTime", 755);
user_pref("browser.slowStartup.samples", 3);
user_pref("browser.startup.homepage", "http://www.schucom.ch/lernstick/");
user_pref("browser.startup.homepage_override.buildID", "20181002222838");
user_pref("browser.startup.homepage_override.mstone", "60.2.2");
user_pref("browser.toolbarbuttons.introduced.pocket-button", true);
user_pref("browser.uiCustomization.state", "{\"placements\":{\"widget-overflow-fixed-list\":[],\"PersonalToolbar\":[\"personal-bookmarks\"],\"nav-bar\":[\"back-button\",\"forward-button\",\"stop-reload-button\",\"home-button\",\"customizableui-special-spring1\",\"urlbar-container\",\"search-container\",\"customizableui-special-spring2\",\"downloads-button\",\"library-button\",\"loop-button\",\"sidebar-button\"],\"TabsToolbar\":[\"tabbrowser-tabs\",\"new-tab-button\",\"alltabs-button\"],\"toolbar-menubar\":[\"menubar-items\"]},\"seen\":[\"loop-button\",\"pocket-button\",\"developer-button\"],\"dirtyAreaCache\":[\"PersonalToolbar\",\"nav-bar\",\"PanelUI-contents\",\"addon-bar\",\"TabsToolbar\",\"toolbar-menubar\"],\"currentVersion\":14,\"newElementCount\":2}");
user_pref("browser.urlbar.daysBeforeHidingSuggestionsPrompt", 3);
user_pref("browser.urlbar.lastSuggestionsPromptDate", 20170224);
user_pref("browser.urlbar.matchBuckets", "general:5,suggestion:Infinity");
user_pref("browser.urlbar.placeholderName", "Google");
user_pref("datareporting.policy.dataSubmissionPolicyAcceptedVersion", 2);
user_pref("datareporting.policy.dataSubmissionPolicyNotifiedTime", "1466876149243");
user_pref("datareporting.sessions.current.activeTicks", 6);
user_pref("datareporting.sessions.current.clean", true);
user_pref("datareporting.sessions.current.firstPaint", 1600);
user_pref("datareporting.sessions.current.main", 41);
user_pref("datareporting.sessions.current.sessionRestored", 1656);
user_pref("datareporting.sessions.current.startTime", "1491476392638");
user_pref("datareporting.sessions.current.totalTime", 26);
user_pref("datareporting.sessions.currentIndex", 13);
user_pref("datareporting.sessions.previous.10", "{\"s\":1487951088518,\"a\":43,\"t\":232,\"c\":true,\"m\":31,\"fp\":1470,\"sr\":1505}");
user_pref("datareporting.sessions.previous.11", "{\"s\":1491476197128,\"a\":2,\"t\":13,\"c\":true,\"m\":777,\"fp\":3881,\"sr\":7255}");
user_pref("datareporting.sessions.previous.12", "{\"s\":1491476279151,\"a\":9,\"t\":43,\"c\":true,\"m\":31,\"fp\":2220,\"sr\":1906}");
user_pref("datareporting.sessions.previous.4", "{\"s\":1487944656593,\"a\":13,\"t\":131,\"c\":true,\"m\":699,\"fp\":3562,\"sr\":6278}");
user_pref("datareporting.sessions.previous.5", "{\"s\":1487945251795,\"a\":5,\"t\":22,\"c\":true,\"m\":30,\"fp\":2052,\"sr\":1797}");
user_pref("datareporting.sessions.previous.6", "{\"s\":1487949098858,\"a\":1,\"t\":6,\"c\":true,\"m\":12,\"fp\":1800,\"sr\":1339}");
user_pref("datareporting.sessions.previous.7", "{\"s\":1487949795728,\"a\":15,\"t\":443,\"c\":true,\"m\":40,\"fp\":1371,\"sr\":1431}");
user_pref("datareporting.sessions.previous.8", "{\"s\":1487950372247,\"a\":1,\"t\":7,\"c\":true,\"m\":80,\"fp\":1383,\"sr\":1430}");
user_pref("datareporting.sessions.previous.9", "{\"s\":1487951036278,\"a\":3,\"t\":13,\"c\":true,\"m\":31,\"fp\":1557,\"sr\":1593}");
user_pref("datareporting.sessions.prunedIndex", 3);
user_pref("devtools.onboarding.telemetry.logged", true);
user_pref("distribution.iniFile.exists.appversion", "60.2.2");
user_pref("distribution.iniFile.exists.value", false);
user_pref("dom.apps.reset-permissions", true);
user_pref("dom.mozApps.used", true);
user_pref("e10s.rollout.cohort", "unsupportedChannel");
user_pref("experiments.activeExperiment", false);
user_pref("extensions.blocklist.pingCountVersion", -1);
user_pref("extensions.databaseSchema", 25);
user_pref("extensions.e10s.rollout.hasAddon", false);
user_pref("extensions.e10sBlockedByAddons", false);
user_pref("extensions.getAddons.cache.lastUpdate", 1491476202);
user_pref("extensions.getAddons.databaseSchema", 5);
user_pref("extensions.lastAppBuildId", "20181002222838");
user_pref("extensions.lastAppVersion", "60.2.2");
user_pref("extensions.lastPlatformVersion", "60.2.2");
user_pref("extensions.pendingOperations", false);
user_pref("extensions.shownSelectionUI", true);
user_pref("extensions.systemAddonSet", "{\"schema\":1,\"addons\":{}}");
user_pref("extensions.ui.dictionary.hidden", true);
user_pref("extensions.ui.experiment.hidden", true);
user_pref("extensions.ui.lastCategory", "addons://list/plugin");
user_pref("extensions.ui.locale.hidden", false);
user_pref("extensions.webextensions.uuids", "{\"screenshots@mozilla.org\":\"20fcda77-699f-4c66-b9fe-9f6552d6de79\"}");
user_pref("gecko.buildID", "20160608013412");
user_pref("gecko.handlerService.migrated", true);
user_pref("gecko.mstone", "47.0");
user_pref("idle.lastDailyNotification", 1487950211);
user_pref("intl.accept_languages", "de-ch,de-de,en-us,en");
user_pref("media.gmp-manager.buildID", "20161213214853");
user_pref("media.gmp-manager.lastCheck", 1487944723);
user_pref("media.gmp.storage.version.observed", 1);
user_pref("network.cookie.prefsMigrated", true);
user_pref("network.predictor.cleaned-up", true);
user_pref("pdfjs.enabledCache.initialized", true);
user_pref("pdfjs.enabledCache.state", true);
user_pref("pdfjs.migrationVersion", 2);
user_pref("pdfjs.previousHandler.alwaysAskBeforeHandling", true);
user_pref("pdfjs.previousHandler.preferredAction", 4);
user_pref("places.database.lastMaintenance", 1487950211);
user_pref("places.history.expiration.transient_current_max_pages", 104858);
user_pref("plugin.disable_full_page_plugin_for_types", "application/pdf");
user_pref("plugin.importedState", true);
user_pref("plugin.state.flash", 2);
user_pref("plugin.state.npica", 2);
user_pref("pref.browser.homepage.disable_button.current_page", false);
user_pref("privacy.cpd.offlineApps", true);
user_pref("privacy.sanitize.migrateFx3Prefs", true);
user_pref("privacy.sanitize.pending", "[]");
user_pref("privacy.sanitize.timeSpan", 0);
user_pref("security.disable_button.openCertManager", false);
user_pref("security.sandbox.content.tempDirSuffix", "b861d6b3-cd8f-4968-a108-90b72e7797af");
user_pref("services.blocklist.addons.checked", 1487895764);
user_pref("services.blocklist.clock_skew_seconds", 54392);
user_pref("services.blocklist.last_etag", "\"1487895540022\"");
user_pref("services.blocklist.last_update_seconds", 1487895764);
user_pref("services.blocklist.onecrl.checked", 1487895764);
user_pref("services.blocklist.plugins.checked", 1487895764);
user_pref("services.sync.clients.lastSync", "0");
user_pref("services.sync.clients.lastSyncLocal", "0");
user_pref("services.sync.declinedEngines", "");
user_pref("services.sync.globalScore", 0);
user_pref("services.sync.migrated", true);
user_pref("services.sync.nextSync", 0);
user_pref("services.sync.tabs.lastSync", "0");
user_pref("services.sync.tabs.lastSyncLocal", "0");
user_pref("signon.importedFromSqlite", true);
user_pref("storage.vacuum.last.index", 0);
user_pref("storage.vacuum.last.places.sqlite", 1487950211);
user_pref("toolkit.startup.last_success", 1544741889);
user_pref("toolkit.telemetry.cachedClientID", "a0fd93df-0540-45fc-ad0f-62dbaf4705c2");
user_pref("toolkit.telemetry.previousBuildID", "20181002222838");
user_pref("toolkit.telemetry.reportingpolicy.firstRun", false);
